contributors/py-helix-admin/helix/cluster.py

```diff
@@ -48,15 +52,13 @@ def __str__(self):
                                            self.cluster)
 
     def __repr__(self):
-        return "{0}({1}, {2})".format(self.__class__.__name__, self.cluster,
-                                      self.host)
+        return "{0}({1}, {2})".format(self.__class__.__name__, self.cluster)
 
     def load_resources(self):
         """queries helix for resource groups and loades them into model"""
         try:
-            for cur_resource in functions.get_resource_groups(self.host,
-                                                              self.cluster):
-                data = functions.get_resource_group(self.host, self.cluster,
+            for cur_resource in self.functions.get_resource_groups(self.cluster):
+                data = self.functions.get_resource_group(self.cluster,
                                                     cur_resource)
```

Comment:
nit: argument after line break doesn't seem to align with arguments on the first line